deps/Makefile

```diff
@@ -116,3 +116,7 @@ jemalloc: .make-prerequisites
 	cd jemalloc && $(MAKE) lib/libjemalloc.a
 
 .PHONY: jemalloc
+
+all: distclean hiredis linenoise hdr_histogram fpconv lua jemalloc
```

Comment:
Jemalloc is only needed if redis is compiled for using jemalloc. The logic for this is in src/Makefile. Look for the variable `DEPENDENCY_TARGETS`.

Reply:
Removed all rule from here as this should depends on DEPENDENCY_TARGETS variable.